src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -144,11 +143,12 @@ function AnnotationShareControl({
                 readOnly
                 ref={inputRef}
               />
-              <Button
+              <IconButton
+                className="InputButton"
                 icon="copy"
                 title="Copy share link to clipboard"
                 onClick={copyShareLink}
-                className="annotation-share-panel__icon-button"
```

Comment:
I still see a rule for 

`&__icon-button` in AnnotationsSharedControl.scss, is it needed?

Reply:
Great catch, thanks!